canvas.go

```diff
@@ -7,6 +7,7 @@ type Canvas interface {
 	SetContent(CanvasObject)
 	Refresh(CanvasObject)
 	Focus(Focusable)
+	Unfocus()
```

Comment:
I had not thought this would be needed.. but looking at the code it seems like it is the right way to do it